### ISSUE TITLE
Can have alphanumberic and special characters together?

### ISSUE THREAD

**Issue by pankaj-mahendran** [klughammer/node-randomstring#42]
Hi,
I need random strings with alphanumberic and special characters (._-,~) as characters set

**Comment by eliaskg** [comment]
Hi,

Thank you for the input. Combined charsets would be a good idea. I will look into this.

Right now my idea would be to have an additional "charsets:" parameter that takes an array of charsets as input.

**Comment by eliaskg** [comment]
Just published 1.3.0 which has the ability to combine character sets. In your case that would be

```
randomstring.generate({
  charset: ['alphanumeric', '._-,~']
});
```
